models/mesh.go

```diff
@@ -27,6 +27,9 @@ type ControlPlane struct {
 	// Config
 	Config ControlPlaneConfiguration
 
+	// Config Map
+	ConfigMap map[string]string `yaml:"configMap,omitempty"`
```

Comment:
Same comment here about just moving the raw configmap to the `ControlPlaneConfiguration` object so that all the config can be under a single struct and you don't have to return multiple values but if you'd rather keep it the way it is that's ok too.